<launch>

    <node pkg="ros_pathtracking" type="ros_pathtracking" name="ros_pathtracking_node" output="screen" required="true">
        <param name="laser_topic" type="string" value="/scan" />
        <param name="cmd_vel" type="string" value="/cmd_vel" />
        <param name="pannel_id" type="string" value="/pannel" />
        <param name="base_id" type="string" value="/base_link" />
        <param name="movebase_polygon" type="string" value="[[0.66,-0.41],[0.66,0.41],[-0.66,0.41],[-0.66,-0.41]]" />
	<!-- <param name="movebase_polygon" type="string" value="[[0.46,-0.31],[0.46,0.31],[-0.46,0.31],[-0.46,-0.31]]" /> -->
    </node>




</launch>
